public/index.html

```diff
@@ -29,7 +29,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Isomer CMS</title>
```

Comment:
why do we not have the space? Isomer and CMS are 2 different words right?

Reply:
Yup, but to be consistent with our comms - see [this thread](https://opengovproducts.slack.com/archives/C04S8QA6ES3/p1693875975226159)